tests/libcxx/expected_results.txt

```diff
@@ -1262,6 +1253,36 @@ std/atomics/atomics.ref/operator_plus_equals.pass.cpp SKIPPED
 std/atomics/atomics.ref/store.pass.cpp SKIPPED
 std/atomics/atomics.ref/wait.pass.cpp SKIPPED
 
+# Not analyzed. Attempting to dereference `void *`.
+std/utilities/memory/unique.ptr/noexcept_operator_star.compile.pass.cpp FAIL
+std/utilities/smartptr/unique.ptr/unique.ptr.class/unique.ptr.observers/dereference.single.pass.cpp FAIL
+
+# Not analyzed. Attempting to delete `nullptr_t`.
+std/utilities/memory/util.smartptr/util.smartptr.shared/util.smartptr.shared.const/pointer_deleter.pass.cpp FAIL
+
+# Not analyzed. Various assertions.
+std/time/time.zone/time.zone.timezone/time.zone.members/get_info.local_time.pass.cpp FAIL
+std/time/time.zone/time.zone.timezone/time.zone.members/to_sys_choose.pass.cpp FAIL
+std/time/time.zone/time.zone.zonedtime/time.zone.zonedtime.members/get_info.pass.cpp FAIL
+std/time/time.zone/time.zone.zonedtime/time.zone.zonedtime.nonmembers/ostream.pass.cpp FAIL
+
+# Not analyzed. `span<volatile string>` causes constraint failure in `std::const_iterator`.
+std/containers/views/views.span/span.cons/span.pass.cpp FAIL

```

Comment:
Should be listed below line 94 as it failed for the same reason.
(I reported LWG-3813 for this, but haven't come up with any resolution yet.)
https://github.com/microsoft/STL/blob/d2fa3a3cf02f09e2396a65c076be8be4c26352e9/tests/libcxx/expected_results.txt#L93-L94